package.json

```diff
@@ -44,9 +45,9 @@
     "cdn:watch:extension": "jung -r ./src -F '.*\\.test\\.js' --run -- npm run cdn:build:extension",
     "cdn:webpack": "npx webpack --progress --config ./webpack.config.js",
     "packages:bundle": "webpack -c tools/jil/webpack.modular.js",
-    "test-server": "node ./tools/jil/bin/server",
-    "sauce:connect": "node ./tools/jil/bin/sauce",
-    "sauce:get-browsers": "node ./tools/jil/util/sauce-browsers",
+    "test-server": "node ./tools/wdio/bin/server",
```

Comment:
Commenting this specifically for brevity other reviewers, running `test-server` will point at the wdio server by default and you will need to manually run `node tools/jil/...` to run the jil test server if needed

Reply:
Oops I forgot to change that back to jil. It's the same server though, just through a different set of cli js files. Should work exactly the same. Thanks for catching that. Should I change it back?

Reply:
I see it's using test-server under the hood, but just for clarity for now, how about adding a script like "wdio-test-server" and restoring the old one?